content/applications/finance/accounting/taxes/taxcloud.rst

```diff
@@ -2,9 +2,19 @@
 TaxCloud integration
 ====================
 
+.. warning::
+   ⚠️ Upgrade warning ⚠️
+
+   The Odoo TaxCloud integration has begun its decommissioning, starting in Odoo 17. New
+   installations are prohibited in Odoo 17. In Odoo 18, the TaxCloud module(s) will **not** exist at
+   all. Odoo recommends the use of the Avatax platform, instead.
+
+   .. seealso::
+      :doc:`avatax`
+
```

Comment:
Pull the seealso admonition out of the warning to be consistent with the previous PR https://github.com/odoo/documentation/pull/5976
```suggestion
.. seealso::
   :doc:`avatax`
```